astropy/io/tests/test_registry.py

```diff
@@ -506,27 +505,16 @@ class MTable(Table):
         mt['a'].format = '.4f'
         mt['a'].description = 'hello'
 
-        if HAS_YAML:
-            ctx = nullcontext()
```

Comment:
lint: since this change, `nullcontext` is no longer used in this module, so its import can be removed.

Reply:
Done!